etc/fetchers/fetch-sheets.js

```diff
@@ -0,0 +1,78 @@
+const cheerio = require("cheerio");
+const fs = require("fs");
+const path = require("path");
+const fetch = require("node-fetch");
+
+function toSnakeCase(text) {
+  return text.trim().toLowerCase().replace(" ", "_");
+}
+function allIsEmptyString(array) {
+  for (const item of array) {
+    if (item.length !== 0) {
+      return false;
+    }
+  }
+  return true;
+}
+
+module.exports.fetchSheets = async function fetchSheets() {
+  const source = await fetch(
+    "https://docs.google.com/spreadsheets/d/1RIcSiQqPCw-6H55QIYwblIQDPpFQmDNC73ukFa05J7c/htmlview"
```

Comment:
```suggestion
    "https://kcov.id/wbw-sheets"
```

Please update the URL to use this shortened one instead, so it's easier to change when we need to change it in the future.

Reply:
Noted, just to confirm, the kcov.id/wbw-sheets points to the `htmlview` version, correct? Because previously it redirects me to the editable version of the sheets.

Reply:
Fixes at ffe2fd1

Reply:
Yes, it is. I have updated the link yesterday. 😅